llvm/lib/Transforms/Scalar/LICM.cpp

```diff
@@ -2234,7 +2241,7 @@ bool llvm::promoteLoopAccessesToScalars(
   LoopPromoter Promoter(SomePtr, LoopUses, SSA, ExitBlocks, InsertPts,
                         MSSAInsertPts, PIC, MSSAU, *LI, DL, Alignment,
                         SawUnorderedAtomic, AATags, *SafetyInfo,
-                        StoreSafety == StoreSafe);
+                        StoreSafety == StoreSafe, StoreIsGuanteedToExecute);
```

Comment:
Rather than passing down the variable, can we pass empty AATags here if !guaranteed?

Reply:
Done, thanks!